src/views/insights/Home/index.vue

```diff
@@ -49,10 +51,20 @@ export default {
     InsightsLayout,
   },
 
+  data: () => ({
+    filterDate: {
+      start: moment().subtract(1, 'day').format('YYYY-MM-DD'),
+      end: moment().format('YYYY-MM-DD'),
+    },
+  }),
+
   methods: {
     goToDashboards() {
       this.$router.replace({ name: 'dashboards' });
     },
+    getTime(date) {
```

Comment:
Where is this being used?

Reply:
This function is not being used so I removed it.